homeassistant/components/ozw/climate.py

```diff
@@ -213,7 +213,11 @@ def current_temperature(self):
     def hvac_action(self):
         """Return the current running hvac operation if supported."""
         if not self.values.operating_state:
-            return None
+            # Thermostat(valve) with no support for hvac actions are considered heating-only
+            if self.current_temperature <= self.target_temperature:
+                return CURRENT_HVAC_HEAT
+            else:
+                return CURRENT_HVAC_IDLE
```

Comment:
We don't calculate fake states in device integrating integrations. The device must report the value for us to set it in the state machine.

Reply:
Fair enough. But the device does not have a state, except for the current and target temperatures. Can you point me in the right direction so that we can fix #28834?

Can you explain the difference with the `hvac_mode` to me? Which is also set by the integration.

Reply:
`hvac_mode` is required to be set as it defines the main state of the entity. We also know what the hvac_mode is if we know that the device can only heat, so we're not setting a fake state.

Reply:
Please open a new issue specific for ozw integration and provide the mqtt dump for the device.
https://www.home-assistant.io/docs/mqtt/service/#service-mqttdump

If the device reports its action we could try to address this. Otherwise there's nothing we can do since we rely on the report from the device.

Reply:
There are already plenty of usable MQTT dumps for this. Even included in the Home Assistant Core tests. 

The device doesn't provide an action, only a setpoint. But I will create a new issue for this.

Reply:
Ok, well then there's nothing we can do and no need for a new issue.